sql/4_1_2-to-4_2_0_upgrade.sql

```diff
@@ -2856,6 +3020,36 @@ UPDATE `clinical_rules` SET `amc_2014_flag` = 1 , `amc_code_2014` = '170.314(g)(
+('LBFlabs', 'gyn', '1Lab Request', 'GYN Cytology', 180, 21, 1, 1, 30, 'Lab_GYN_Cytology', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'hematology', '1Lab Request', 'Hematology', 170, 21, 1, 1, 30, 'Lab_Hematology', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'icd_4', '1Lab Request', 'ICD Codes', 140, 15, 1, 40, 45, '', 1, 1, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'insur', '1Lab Request', 'Insurance Name', 110, 2, 1, 30, 30, '', 1, 1, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'insur_group', '1Lab Request', 'Insurance Group', 130, 2, 1, 30, 30, '', 1, 1, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'insur_id', '1Lab Request', 'Insurance ID', 120, 2, 1, 30, 30, '', 1, 1, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'micro', '1Lab Request', 'Microbiology', 190, 21, 1, 1, 30, 'Lab_Microbiology', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'Other1', '1Lab Request', 'Other Tests', 210, 21, 1, 1, 30, 'Lab_Other_Tests', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'other2', '1Lab Request', '', 220, 21, 1, 1, 30, 'Lab_Other_Test_2', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'Other3', '1Lab Request', 'Other', 230, 3, 1, 45, 180, '', 1, 0, '', '', '', 3, '', 'F', ''),
+('LBFlabs', 'panel', '1Lab Request', 'Panel', 160, 21, 1, 1, 30, 'Lab_Panel', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'spacer2', '1Lab Request', '', 200, 0, 1, 0, 1, '', 1, 0, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'spacer_1', '1Lab Request', '', 150, 0, 1, 0, 1, '', 1, 3, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'tech1', '1Lab Request', 'Technician', 100, 2, 1, 15, 15, '', 1, 1, '', '', '', 0, '', 'F', ''),
+('LBFlabs', 'timestamp', '1Lab Request', 'Time Collected', 90, 2, 1, 15, 15, '', 1, 1, '', '', '', 0, '', 'F', '');
+#EndIf
+
+#IfNotRow2D layout_options form_id LBFPHEX field_id breast
+INSERT INTO `layout_options` (`form_id`, `field_id`, `group_name`, `title`, `seq`, `data_type`, `uor`, `fld_length`, `max_length`, `list_id`, `titlecols`, `datacols`, `default_value`, `edit_options`, `description`, `fld_rows`, `list_backup_id`, `source`, `conditions`) VALUES
+('LBFPHEX', 'breast', '1Normal', 'Breasts', 60, 25, 1, 55, 0, 'PHEXGEN6', 10, 10, '', '', '', 0, '', 'F', ''),
+('LBFPHEX', 'gen', '1Normal', 'General', 10, 25, 1, 55, 0, 'PHEXGEN', 10, 10, '', '', '', 0, '', 'F', ''),
+('LBFPHEX', 'heart', '1Normal', 'Heart', 40, 25, 1, 55, 0, 'PHEXGEN4', 10, 10, '', '', '', 0, '', 'F', ''),
+('LBFPHEX', 'heent', '1Normal', 'HEENT', 20, 25, 1, 55, 0, 'PHEXGEN2', 10, 10, '', '', '', 0, '', 'F', ''),
+('LBFPHEX', 'lungs', '1Normal', 'Lungs', 50, 25, 1, 55, 0, 'PHEXGEN5', 10, 10, '', '', '', 0, '', 'F', ''),
+('LBFPHEX', 'neck', '1Normal', 'Neck/Nodes', 30, 25, 1, 55, 0, 'PHEXGEN3', 10, 10, '', '', '', 0, '', 'F', '');
+#EndIf
+
 #IfNotRow2D layout_options form_id HIS field_id dc_father
```

Comment:
My initial instinct is that using LBF forms seems a bit awkard, but will see what other developers think.